### PR TITLE
Draft 0.74 Release Notes Yoga Snippet

### PR DESCRIPTION
**Pull request by NickGerleman**
`position: "static"` part is WIP.

image

### CODE REVIEW

File: website/blog/2024-04-15-0.74-post.md [hunk 1]
```diff
@@ -28,9 +29,105 @@ Today we're releasing React Native 0.74!...
   - Contextualize what changes are coming into 0.74 from proposal
 - Warning on using react-native init https://github.com/facebook/react-native/commit/a6397afb7f4279e70afa025fe6f9fa0b8e3de3fd
 
-## Yoga 3.0
+## New Layout behaviors
```
Comment (NickGerleman) [inline website/blog/2024-04-15-0.74-post.md:32]:
```suggestion
## New layout behaviors
```

File: website/blog/2024-04-15-0.74-post.md [hunk 2]
```diff
@@ -28,9 +30,94 @@ Today we're releasing React Native 0.74!...
...
+![alignContent references](../static/blog/assets/0.74-align-content.png)
+</a>
+
+## New Architecture support for `position: 'static'`
+
+Elements marked as [`position: 'static'`](https://developer.mozilla.org/en-US/docs/Web/CSS/position#static) may not be offset, and are not considered when determining the [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) of an absolutely positioned element. This allows positioning an element relative to an ancestor which is not its direct parent.
+
+[Picture?]
+
+`position: 'static'` may also affect [z-ordering](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) of views. [@joevilches could give better description]
```
Comment (NickGerleman) [inline website/blog/2024-04-15-0.74-post.md:116]: cc @joevilches

File: website/blog/2024-04-15-0.74-post.md [hunk 3]
```diff
@@ -28,9 +30,94 @@ Today we're releasing React Native 0.74!...
   - Contextualize what changes are coming into 0.74 from proposal
 - Warning on using react-native init https://github.com/facebook/react-native/commit/a6397afb7f4279e70afa025fe6f9fa0b8e3de3fd
 
-## Yoga 3.0
+## New layout behaviors
 
-- Link to Yoga post, need clarification on how it affects users.
+React Native 0.74 includes [changes to Yoga](https://yogalayout.dev/blog/announcing-yoga-3.0) which enable more correct layout. These changes make styling more predictable, and enable rendering components written for the web with greater fidelity.
+
+React Native intentionally preserves some incorrect layout behaviors, where fixing them was found to effect a signigicant number of real-world components. Layout conformance will be able to be configured more granularly in future versions of React Native.
```
Comment (NickGerleman) [inline website/blog/2024-04-15-0.74-post.md:37]: I debated including this part:

> Bleeding edge users on the New Architecture can test the effects of rendering their components in a mode which prioritizes correctness and Web compatibility over compatibility with existing React Native components. This API will change in a future version of React Native.
>
> ```tsx
> import {} from 'react-native/types/experimental'
> 
> // This component tree inherits layout which acts like web
> <View experimental_layoutConformance="strict">
>   <Widget />
> </View>
> 
> // This component tree inherits layout which acts like React Native
> <View experimental_layoutConformance="classic">
>   <Doodad />
> </View>
> ```

Comment (lunaleaps) [reply website/blog/2024-04-15-0.74-post.md:37]: I think that might be interesting to add in a "Tip" clause? Or perhaps an "Info" admonition https://docusaurus.io/docs/next/markdown-features/admonitions

Comment (NickGerleman) [reply website/blog/2024-04-15-0.74-post.md:37]: I think I will avoid it for now until it can be contextualized better.

**Pull request merged**